platform-service/src/unstract/platform_service/utils.py

```diff
@@ -59,3 +61,12 @@ def format_float_positional(value: float, precision: int = 10) -> str:
     """
     formatted: str = f"{value:.{precision}f}"
     return formatted.rstrip("0").rstrip(".") if "." in formatted else formatted
+
+
+class RemoveAccessLogTimestamp(logging.Filter):
```

Comment:
Why are we doing this @Deepak-Kesavan

Reply:
There was a timestamp hardcoded in the message by 
Werkzeug in the access log and the same is being propagated to gunicorn.access logger. Since we are adding a timestamp from our side, I tried to remove it. 

It worked incase of Werkzeug which is when flask run is used. But when I run with gunicorn, it didn't work as expected. Just left it for now because it is not that much important and we just need access logs for debugging purposes.

Reply:
@Deepak-Kesavan let's remove this filter. Adding a filter if not working might just add performance implications. Also the rgex by it's nature is costly operation. So let's remove this filter all together. Having 2 timestamps is something that we can live with. Adding this filter is probably gonna over engineer things and add unnecessary complexity.